### PR TITLE
Add Russian layout

### PR DESCRIPTION
**Pull request by carrot-cookie**
Greetings. 
I made a Russian layout. It looks like this:
image

In my opinion, the right side is empty, and there we can either move additional keys from the center, or add something new

### CODE REVIEW

File: res/xml/qwerty_ru.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <row>
+    <key key0="й" key1="esc" key2="1" key3="~" key4="!"/>
+    <key key0="ц" key2="2" key3="\@"/>
+    <key key0="у" key2="3" key3="\#"/>
+    <key key0="к" key2="4" key3="$"/>
+    <key key0="е" key1="ё" key2="5" key3="%"/>
+    <key key0="н" key2="6" key3="^"/>
+    <key key0="г" key2="7" key3="&amp;"/>
+    <key key0="ш" key2="8" key3="*"/>
+    <key key0="щ" key1="accent_macron" key2="9" key3="(" key4=")"/>
```
Comment (Julow) [inline res/xml/qwerty_ru.xml:12]: You don't need to include any accents, none of them will have any effects on Russian letters. The `accent_...` keys are only meant for the "programming" layouts, currently only `qwerty`.

File: res/values/arrays.xml [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <item>azerty</item>
     <item>qwerty</item>
     <item>qwerty_lv</item>
+    <item>qwerty_ru</item>
```
Comment (Julow) [inline res/values/arrays.xml:8]: I think this layout isn't named qwerty. Wikipedia calls it [JCUKEN](https://en.wikipedia.org/wiki/JCUKEN).

There's an other layout that would be attributed the name "russian qwerty" ([phonetic layout](https://en.wikipedia.org/wiki/Phonetic_keyboard_layout)) so keep the name free.

File: res/xml/qwerty_ru.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    <key key0="з" key2="0"/>
+    <key key0="х"/>
+    <key key0="ъ"/>
+  </row>
+  <row>
+    <key shift="0.5" key0="ф" key1="tab" key2="`"/>
+    <key key0="ы" key1="accent_ring" key3="ß"/>
```
Comment (Julow) [inline res/xml/qwerty_ru.xml:19]: No need to include `ß` either.

**Review by Julow — COMMENTED** [review]
Thanks a lot!

The symbols keys look stacked on the left, I think it would be better to spread them a little, especially the top row. At the same time, avoiding keys at the left and right edges of the keyboard, which are harder to type.

**Comment by carrot-cookie** [comment]
Fixed the problems above. Here's what happened
image

**Comment by Julow** [comment]
There have been some changes on the master branch so I fixed the conflicts and I renamed the layout.

Thanks a lot for the work :) Merging !

**Pull request merged**